tests/test_collapse.py

```diff
@@ -73,6 +73,7 @@ def test_diff(self):
     def test_reconcile_prefix(self):
         """Test curie reconciliation is performing as expected."""
         msdf = parse_sssom_table(data_dir / "basic3.tsv")
+        msdf.clean_prefix_map()
```

Comment:
we now backfill the prefix map with the default one, so it was simpler to remove the unused parts here rather than changing the test to be explicit